x-pack/legacy/plugins/ml/public/application/timeseriesexplorer/_timeseriesexplorer.scss

```diff
@@ -34,7 +34,9 @@
     padding: $euiSize;
 
     .panel-title {
-      color: $euiTitleColor;
+      font-size: $euiFontSizeM;
+      font-weight: 400;
+      color: $euiColorDarkShade;
```

Comment:
Already spotted `.panel-title` class with similar styles  👀

Reply:
As above, going to stick with `euiFontSizeM` here as it retains the `18px` font size used previously.